features/.nav

```diff
@@ -44,22 +44,21 @@
   - color.feature
   - fail_fast.feature
   - custom_settings.feature
-  - alias_example_to.feature
   - default_path.feature
 - expectation_framework_integration:
-  - configure_expectation_framework.feature
   - failure_aggregation.feature
-- mock_framework_integration:
-  - use_rspec.feature
-  - use_flexmock.feature
-  - use_mocha.feature
-  - use_rr.feature
-  - use_any_framework.feature
 - formatters:
   - text_formatter.feature
   - configurable_colors.feature
   - json_formatter.feature
   - custom_formatter.feature
-- spec_files:
-  - arbitrary_file_suffix.feature
 - core_standalone.feature
+- legacy_support_and_special_project_needs
```

Comment:
So, a few ideas here:

* "Using RSpec with Alternate Tools"
* "Usage with Other Testing Libraries"
* "Usage with Alternate Tools"

Although, all 3 of those ideas would probably not fit `arbitrary_file_suffix` or `alias_example_to` but I think that's OK.  We can find a better place for those two.

Also, I don't think we need `use_rspec_mocks.feature` -- the default behavior is to use rspec-mocks, so why should we take up valuable doc space with this?  We'd need to make sure the examples in that file are covered by specs, of course.